docs/section10/10-7.md

```diff
@@ -86,8 +83,9 @@
                     - Else capacity is >=760,000 Btuh: `Else:`
                         - Get baseline efficiency for the >=760,000 Btuh size category:`COPnf_full_load_760Btuh_b = data_lookup(table_G3_5_1,760Btuh)`
                         - Check if the modeled efficiency is consistent with this value, if so set boolean variable to true: `if full_load_efficiency_b == COPnf_full_load_760Btuh_b: meets_lower_or_equiv_size_category_eff_b = TRUE`  
-                
-                 - Check else if the system type is 4 and that num_zones_defined_for_all_zones equals TRUE (pass if efficiency meets size category associated with the system):`elif (hvac_sys_type_b = "SYS-3" or hvac_sys_type_b = "SYS-3b") AND num_zones_defined_for_all_zones == FALSE:`
+               
+                 - Check else if the system type is 4 and that the associated zone number of aggregated hvac zones was defined (pass if efficiency meets size category associated with the system):`elif (hvac_sys_type_b = "SYS-3" or hvac_sys_type_b = "SYS-3b") AND zone_list_b[0].equipment_count != Null:`
```

Comment:
Code should reference system 4 and 4b, as mentioned in the charging note, not 3 & 3b.

Reply:
Corrected.